libs/ui/src/ui_strings/election_strings.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {
+  Candidate,
+  Contest,
+  Parties,
+  Party,
+  getCandidateParties,
+} from '@votingworks/types';
+
+import { UiString } from './ui_string';
+
+/* istanbul ignore next - mostly presentational, tested via apps where relevant */
+export const electionStrings = {
+  // TODO(kofi): Fill out.
+
+  // NOTE: Omitting 'type' to support both the `Contest` and the
+  // `MsEitherNeitherContest` types.
```

Comment:
Would it be better to do either say which types you're expecting (`Contest | MsEitherNeitherContest`) or declare the shape of the object you need (`{ id: string; title: string }`)?

Reply:
Yeah, good call. I initially wanted slightly stricter typing for the contest, since I thought `{ id, title }` might match some other election objects, but shouldn't be an issue, now that I think about it. (Couldn't go the `Contest | MsEitherNeitherContest` route without additional work, since `MsEitherNeitherContest` is declared in `mark-flow-ui`)